drivetrain.py

```diff
@@ -15,6 +15,12 @@
 
 logger = logging.getLogger(__name__)
 
+# battery voltage check contasts
+BATT_ADC_GAIN = 0.02909

```

Comment:
Is there any reason why these are module level declarations? They're only used in `Drivetrain`, aren't they?